pkg/trace/api/container.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+//go:build !linux
+// +build !linux
+
+package api
+
+import (
+	"context"
+	"net"
+	"net/http"
+)
+
+func connContext(ctx context.Context, c net.Conn) context.Context {
+	// unimplemented for non-linux builds.
+	return ctx
+}
+
+// GetContainerID returns the container ID set by the client in the request header, or the empty
+// string if none is present.
+func GetContainerID(ctx context.Context, h http.Header) string {
```

Comment:
```suggestion
func GetContainerID(_ context.Context, h http.Header) string {
```

Reply:
Do you think we should follow the [general consensus](https://go.dev/doc/effective_go#Getters) of avoiding `Get*` or is it better to have it here?

Reply:
`ContainerID` seemed too non-descript, and couldn't think of a better name. I agree the `Get` prefix isn't good. Open to suggestions.

Reply:
Fine by me to keep it as it is.